APMrover2/control_modes.cpp

```diff
@@ -117,3 +117,14 @@ void Rover::read_trim_switch()
     }
 }
 
+bool Rover::is_moving()
```

Comment:
I was going to add to the todo list a check in here to see if the boat was drifting and not allow disarming.  I had concerns that if the boat was drifting you wouldn't be able to arm it again if you disarmed.
I also had a thought around a boat using its rudder only and no motor to control it - if it was in a current or something and they held the rudder stick to steer, it disarmed but then they wanted to run the motor so they would have to arm again with the rudder NOT in the direction they wanted.  
Actually, I will rename this to motor_active and when I write the code for the situation above is_moving can call motor_active if it needs to.
Thanks!